docs/conf/atlantic/2024/news/welcome.rst

```diff
@@ -0,0 +1,68 @@
+:template: {{year}}/generic.html
+
+.. post:: January 4, 2024
+   :tags: {{shortcode}}-{{year}}
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+Welcome to the official launch of `Write the Docs {{ city }} {{year}} <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/>`_.
+We're excited to announce the dates for our next conference: **Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**.
```

Comment:
```suggestion
We're excited to announce the dates for our next conference: **Write the Docs {{ city }} {{ year }} will be held on {{ date.short }}**.
```

Reply:
I wonder if we should use the fully-spelled-out month name instead of the shortname for the month?